cocos/particle/renderer/particle-system-renderer-gpu.ts

```diff
@@ -162,7 +163,7 @@ export default class ParticleSystemRendererGPU extends ParticleSystemRendererBas
         this._frameTile_velLenScale = new Vec4(1, 1, 0, 0);
         this._unifrom_velLenScale = this._frameTile_velLenScale.clone();
         this._tmp_velLenScale = this._frameTile_velLenScale.clone();
-        this._node_scale = new Vec4();
+        this._node_scale = new Vec3();
```

Comment:
Why change to Vec3 as the type of _node_scale is Vec4.

Reply:
_node_scale  should be a type of Vec3, and actually we use it as Vec3 here
https://github.com/cocos/cocos-engine/blob/899a3a989746f729da792d88cb84c6e32710be59/cocos/particle/renderer/particle-system-renderer-gpu.ts#L293-L305

Reply:
it is because `_particleSystem` is a type of `any` before, so we never encounter this type issue

Reply:
But the declaration of _node_scale is not changed.

Reply:
_node_scale is private